.theme/wysiwyg.css

```diff
@@ -73,8 +69,8 @@ Preview */
 }
 
 .markdown-preview-section > div {
-  padding-left: 4.5rem;
-  padding-right: 3.5rem;
+  padding-left: 3rem;
+  padding-right: 3rem;
```

Comment:
If you could please make the following changes

```
padding-left: 4rem;
padding-right: 2rem;
```

and in line 52, in `.CodeMirror-scroll` section 

`padding-left: 4rem;`

it would take care of the issues i mentioned in https://github.com/radekkozak/discordian/pull/40#issuecomment-801504752